.travis.yml

```diff
@@ -9,7 +9,7 @@ script:
 jobs:
   include:
   - stage: Lint and Test
-    script: npm run lint && npm run test:coveralls && npm run size
+    script: npm run lint && npm run test && npm run size && npm run test:coveralls
```

Comment:
Can we run `npm run lint && npm run test:coverage && npm run size && npm run test:coveralls` and then edit the `npm run test:coveralls` command to just pipe the coverage info to coveralls instead of running the tests again?